middleware/oapi_validate.go

```diff
@@ -1,12 +1,9 @@
-// Package middleware implements middleware function for go-chi or net/http,
-// which validates incoming HTTP requests to make sure that they conform to the given OAPI 3.0 specification.
-// When OAPI validation failes on the request, we return an HTTP/400.
```

Comment:
Why was this package comment removed?

Reply:
I actually rewrote the package, then reduced the diff, forgot about the comment